src/pages/umi/web3js-differences-and-adapters.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Web3.js Differences and Adapters
+metaTitle: Umi - Web3.js Differences and Adapters
+description: Difference and Adapters to make Metaplex Umi work with Solana web3js.
+---
+
+The `@solana/web3.js` library is currently widely used in the Solana ecosystem and defines its own types for `Publickeys`, `Transactions`, `Instructions`, etc. 
+
+When creating `Umi`, we wanted to move away from the class-based types defined in `@solana/web3.js` and instead use a more functional approach by relying only on TypeScript types. This unfortunately means that, although having the same or similar import names, not all types from `@solana/web3.js` are compatible with the ones provided by `Umi` and vice versa.
+
+To help with this issue, `Umi` provides a set of adapters that allows to parse types to and from their `Web3.js` counterparts and they can be found in the [`@metaplex-foundation/umi-web3js-adapters`](https://www.npmjs.com/package/@metaplex-foundation/umi-web3js-adapters) package.
+
+## Required Package and Imports
+
+To install the `umi-web3js-adapters` package and be abel to access to a bunch of helper methods to convert to and from Web3.js types, use the following commands:
+
+```
+npm i @metaplex-foundation/umi-web3js-adapters
```

Comment:
If you have @metaplex-foundation/umi installed you do not need this package.
Quite many people feel (for whatever reason) adding more packages as "not good" (even though they are dependent on each other, or even included like here either way)

Reply:
Oh nice, I thought it was mandatory to install it!